packages/google_maps_flutter/google_maps_flutter/test/fake_maps_controllers.dart

```diff
@@ -345,6 +365,76 @@ class FakePlatformGoogleMap {
     return result;
   }
 
+  Set<HeatmapId> _deserializeHeatmapIds(List<dynamic>? heatmapIds) {
+    if (heatmapIds == null) {
+      return <HeatmapId>{};
+    }
+    return heatmapIds
+        .map(
+          (dynamic heatmapId) => HeatmapId(heatmapId as String),
+        )
+        .toSet();
+  }
+
+  Set<Heatmap> _deserializeHeatmaps(dynamic heatmaps) {
+    if (heatmaps == null) {
+      return <Heatmap>{};
+    }
+    final List<dynamic> heatmapsData = heatmaps as List<dynamic>;
+    final Set<Heatmap> result = <Heatmap>{};
+    for (final Map<dynamic, dynamic> heatmapData
+        in heatmapsData.cast<Map<dynamic, dynamic>>()) {
+      final String heatmapId = heatmapData['heatmapId'] as String;
+
+      final List<dynamic> dataData = heatmapData['data'] as List<dynamic>;
+      final List<WeightedLatLng> data =
+          dataData.map((dynamic e) => WeightedLatLng.fromJson(e)!).toList();
+
+      final bool dissipating = heatmapData['dissipating'] as bool;
+
+      final Map<String, dynamic>? gradientData =
+          heatmapData['gradient'] as Map<String, dynamic>?;
+      final HeatmapGradient? gradient;
+      if (gradientData != null) {
+        final List<Color> colors = (gradientData['colors'] as List<dynamic>)
```

Comment:
There are analysis issues:
```
  error - test/fake_maps_controllers.dart:399:20 - The name 'Color' isn't a type so it can't be used as a type argument. Try correcting the name to an existing type, or defining a type named 'Color'. - non_type_as_type_argument
  error - test/fake_maps_controllers.dart:401:14 - Missing type arguments for generic method 'map<T>'. Try adding an explicit type, or remove implicit-dynamic from your analysis options file. - implicit_dynamic_method
  error - test/fake_maps_controllers.dart:401:29 - The method 'Color' isn't defined for the type 'FakePlatformGoogleMap'. Try correcting the name to the name of an existing method, or defining a method named 'Color'. - undefined_method
  error - test/fake_maps_controllers.dart:411:51 - The 
```

Reply:
What version of flutter is the CI using? These don't appear locally for me on flutter stable.

Reply:
It went away with your rebase.